src/main/kotlin/no/nav/pensjon/pdfmerger/advancedMerge/models/RequestModels.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.pensjon.pdfmerger
+
+data class MergeInfoRequest(
+    val gjelderID: String,
+    val gjelderNavn: String,
+    val dokumentinfo: List<DokumentinfoRequest>
+)
+
+data class DokumentinfoRequest(
+    val filnavn: String,
+    val dokumenttype: String,
+    val fagomrade: String,
+    val saknr: String,
+    val avsenderMottaker: String,
+    val dokumentnavn: String,
+    val mottattSendtDato: String,
+    val vedleggListe: List<VedleggDokumentRequest>?
+)
+
+data class VedleggDokumentRequest(
+    val filnavn: String,
+    val dokumentnavn: String
+)
```

Comment:
Trenger vi egentlig egne klasser for Request?

Reply:
Jeg liker det å ha et skille mellom input og domenelogikken, da får vi også omstrukturert inputen så den er lettere å jobbe med.
Det endte jo her med at jeg blander mapping og validering, men det må jeg se om jeg får skilt fra hverandre bedre.